mappings/net/minecraft/entity/mob/ShulkerEntity.mapping

```diff
@@ -9,11 +10,13 @@ CLASS aqv net/minecraft/entity/mob/ShulkerEntity
 		FIELD b counter I
 	FIELD bA ATTR_COVERED_ARMOR_BONUS_UUID Ljava/util/UUID;
 	FIELD bB ATTR_COVERED_ARMOR_BONUS Laim;
+	FIELD bz COLOR Lpx;
 	FIELD b ATTACHED_FACE Lpx;
 	FIELD c ATTACHED_BLOCK Lpx;
 	FIELD d PEEK_AMOUNT Lpx;
 	METHOD a setPeekAmount (I)V
 	METHOD dM getAttachedFace ()Lex;
 	METHOD dN getAttachedBlock ()Les;
 	METHOD dO getPeekAmount ()I
+	METHOD dT getColor Lawr;
```

Comment:
Shouldn't that be ()Lawr;? How did you make those mappings, exactly? Enigma or a third-party tool?